content/en/docs/Installation/deployment-configuration.md

```diff
@@ -0,0 +1,274 @@
+---
+title: "Deployment Configuration"
```

Comment:
I understand why you put this section here, and not under Configuration, but it may be a little less confusing if we change the title.  How about "Deployment Options"?